lgc/state/PipelineState.cpp

```diff
@@ -1003,10 +1003,12 @@ bool PipelineState::isTessOffChip() {
 void PipelineState::initializePackInOut() {
   // Pack input/output requirements:
   // 1) -pack-in-out option is on
-  // 2) It supports VS-FS, VS-TCS-TES-(FS)
-  if (PackInOut && !m_unlinked && hasShaderStage(ShaderStageVertex) && !hasShaderStage(ShaderStageGeometry)) {
+  // 2) It is a linked pipeline
+  // 2) It supports VS-FS, VS-TCS-TES-(FS), VS-GS-(FS), VS-TCS-TES-GS-(FS)
+  if (PackInOut && !m_unlinked && hasShaderStage(ShaderStageVertex)) {
     const unsigned nextStage = getNextShaderStage(ShaderStageVertex);
-    m_packInOut = nextStage == ShaderStageFragment || nextStage == ShaderStageTessControl;
+    m_packInOut =
```

Comment:
it looks you can support all pipelines now, can you simplify the check and related comments?